### ISSUE TITLE
Document how to make evcxr a rust jupyter kernel on Mac OS X

### ISSUE THREAD

**Issue by Congee** [google/evcxr#17]
I confirm evcxr_jupyter works on my mac box. But I spent more time than expected to make it work because I couldn't find any relevant documentation.

Below is what should be added to the docs.
```
$ brew install zmq
$ evcxr_jupyter --install
Writing /Users/CC/Library/Application Support/jupyter/kernels/rust/kernel.json
Installation complete
$ jupyter  kernelspec install ~/Library/Application\ Support/jupyter/kernels/rust
[InstallKernelSpec] Removing existing kernelspec in /usr/local/share/jupyter/kernels/rust
[InstallKernelSpec] Installed kernelspec rust in /usr/local/share/jupyter/kernels/rust
```

**Comment by kbarros** [comment]
Alternatively, if you

```mv /Users/$USER/Library/Application\ Support/jupyter/kernels/rust /Users/$USER/Library/Jupyter/kernels/```

then it is not necessary to run the `jupyter  kernelspec install` command.

**Comment by Congee** [comment]
FYI, I am using jupyter 4.4.0 and Mac OS X 10.11.6

**Comment by davidlattimore** [comment]
Thanks for filing this bug. I think this is fixed by #22. Feel free to reopen, or send a PR if you think there's anything else missing.

**Comment by rootex-** [comment]
> I confirm evcxr_jupyter works on my mac box. But I spent more time than expected to make it work because I couldn't find any relevant documentation.
> 
> Below is what should be added to the docs.
> 
> ```
> $ brew install zmq
> $ evcxr_jupyter --install
> Writing /Users/CC/Library/Application Support/jupyter/kernels/rust/kernel.json
> Installation complete
> $ jupyter  kernelspec install ~/Library/Application\ Support/jupyter/kernels/rust
> [InstallKernelSpec] Removing existing kernelspec in /usr/local/share/jupyter/kernels/rust
> [InstallKernelSpec] Installed kernelspec rust in /usr/local/share/jupyter/kernels/rust
> ```

Thanks to @Congee, this helped me to configure rust jupyter for jupyter notebook. 

@davidlattimore, still a bug. Instructions mentioned here https://github.com/google/evcxr/blob/main/evcxr_jupyter/README.md 
were not enough to make jupyter notebook work with Rust.
